packages/cycle-scripts/template/config/JavaScript/flavor.js

```diff
@@ -2,10 +2,6 @@ const dependencies = {
   basics: [
     '@cycle/dom@17.1.0'
   ],
-  language: {
```

Comment:
yes, but as this file is splittet you can add them to the basics array. We could split the file, so the basics, and stream lib stuff is common and there is an additional entry in the `flavor.js` for language specific imports